osprey.js

```diff
@@ -32,9 +32,12 @@ exports.addJsonSchema = function (schema, key) {
 exports.loadFile = function (path, opts) {
   var options = opts || {}
 
-  return require('raml-parser')
-    .loadFile(path)
-    .then(function (raml) {
+  return require('raml-1-parser')
+    .loadRAML(path)
+    .then(function (ramlApi) {
+      var raml = ramlApi.toJSON({
```

Comment:
The `toJSON` method won't return types and schemas included in libraries (included with the `uses` keyword). You will need to recursively obtain the libraries (with the `uses()` method) and add the types and schemas included there with the corresponding namespace.